models/Category.js

```diff
@@ -12,7 +12,7 @@ Category.init(
       primaryKey: true,
       autoIncrement: true,
     },
-    catagory_name: {
+    category_name: {
```

Comment:
😄

Reply:
Must have overlooked that a thousand times. thanks for looking !